nipy/algorithms/clustering/tests/test_gmm.py

```diff
@@ -110,7 +111,7 @@ def test_em_selection():
     dim = 2
     x = np.concatenate((nr.randn(100, dim), 3 + 2 * nr.randn(100, dim)))
 
-    krange = range(1, 10)
+    krange = list(range(1, 10))
```

Comment:
Same for these guys - in each case you're right - the list gets used inside the function as a iterable, but the docstring of the function says it should be a list.   Later on, I guess someone could change the implementation so that it does actually need a sequence, and then this would either break, or something quiet and horrible would happen.

The general idea is that - I would prefer to pass a list, unless an iterable is specifically stated to be acceptable.    Seem reasonable?